tests/sqlalchemy_test_suite/test_suite.py

```diff
@@ -0,0 +1,499 @@
+#!/usr/bin/env python
```

Comment:
These should not be necessary, please remove them everywhere

Reply:
I've put the changes in another PR: https://github.com/snowflakedb/snowflake-sqlalchemy/pull/306